plugins/action_generate_document/src/generateDoc.ts

```diff
@@ -13,7 +13,10 @@ const generateDoc: ActionPluginType = async ({
     parameters
   const userId = parameters?.userId ?? applicationData?.userId
   const applicationSerial = parameters?.applicationSerial ?? applicationData?.applicationSerial
-  const templateId = parameters?.templateId ?? applicationData?.templateId
+  // We don't want to include template id normally, as that will link the file
+  // to the template when exporting, which we only want for Carbone docs and the
+  // like. So we'll only use "templateId" if it's explicitly provided.
+  const templateId = parameters?.templateId
```

Comment:
This prevents all the output docs getting the template_id saved, otherwise we copy every single output doc when exporting a template, which we definitely *don't* want.

This won't fix *existing* files though, so if you export a template that already has output docs associated with it, they'll still come along for the ride (just delete them manually from the export zip I guess)